src/engine/strat_engine/filesystem.rs

```diff
@@ -153,6 +153,21 @@ pub fn create_fs(dev_path: &Path) -> EngineResult<()> {
     Ok(())
 }
 
+pub fn grow_fs(dev_path: &Path) -> EngineResult<()> {
```

Comment:
At some point, I was given to understand that these ```*_fs``` functions were junk functions that would be eliminated at some time in the future. Are they still junk functions or are they expected to be as much permanent parts of our code base as the other parts?